### ISSUE TITLE
found some inconsistencies (album included)

### ISSUE THREAD

**Issue by pcuinty** [gravitystorm/openstreetmap-carto#2222]
While randomly browsing South America, I found some unexpected behaviours. I made some [screenshots with description](https://imgur.com/a/DVnxB)

**Comment by math1985** [comment]
Could this be a problem in the data?

**Comment by pcuinty** [comment]
no idea. I'm just a user of osm with little technical knowledge

**Comment by imagico** [comment]
This is partly #1391, partly the use of Natural Earth data at the lowest zooms.  I am not sure why Montevideo vanishes - IIRC the labeling priorities among cities are the same at the different zoom levels.

**Comment by nebulon42** [comment]
Yes, that's it. NE is responsible for the unclosed border, the Bogotá issue is the result of #1391. I think that the disappearance of Montevideo is the result of a collision between the labels of Buenos Aires and Montevideo.

I'm not really concerned by "Some countries have labels some don't". After #1989 we have more country labels than before and only Suriname and French Guyana are not labelled, because there is not enough space. Maybe we could add some form of displacement strategy if that is supported by Mapnik.

@pcuinty Thanks for taking the time to compile this gallery!

**Comment by imagico** [comment]
> the disappearance of Montevideo is the result of a collision between the labels of Buenos Aires and Montevideo

That might be possible - Buenos Aires probably has higher priority than Montevideo but is not shown at the lower zoom due to collision with Uruguay but at the higher zoom this is not the case so it is shown and prevents the Montevideo label from showing.

> Should I report to Natural Earth?

No, this is not an error, this is just a mismatch between the data sets.

**Comment by StyXman** [comment]
I you're referring to the white region *between* BsAs and Montevideo, it's because the Río de la Plata is a river and not rendered until higher zoom levels.

On the other hand, weren't capitals rendered before anything else? Or is that after we switch from NE data to OSM further in? I've been thinking lately about having some extracts or maybe more small tables with low zoom level data, like `planet_osm_roads`, as far as i understand...

**Comment by kocio-pl** [comment]
This issue is too general, so let's make a summary:
- [x] Uruguay border is still broken, but it looks like a data issue, so doesn't belong here
- [x] capitals and big cities visibility is much improved lately
- [x] country/region labels are a duplicate of some other issues

I've found the problem of water area on z<=5, but this looks like a part of https://github.com/gravitystorm/openstreetmap-carto/issues/2293, so I'm going to close this issue in a few days. 

If there are some more specific problems, please consider opening separate tickets for them.

**Comment by skorasaurus** [comment]
@kocio-pl - mind sharing any particular commits that you think improve it? 

(may think about tackling this for the HOT HDM-CartoCSS if it's feasible.).

**Comment by kocio-pl** [comment]
I'm not sure what exactly do you mean, could you elaborate on that?
